doc/usage/configuration.rst

```diff
@@ -393,9 +401,19 @@ General configuration
 
       Added ``i18n.inconsistent_references``
 
-    .. versionadded:: 7.1
+   .. versionadded:: 7.1
+
+      Added ``index`` warning type.
+
+   .. versionchanged:: 7.2
+
+      Make ``toc``, ``i18n`` and ``index`` warning types cover all relative
+      warnings.
+
+      Added ``i18n.babel``, ``i18n.not_readable``, ``i18n.not_writeable``,
```

Comment:
What would those warnings mean?

Actually we should maybe have a follow-up issue which documents where and when warnings of such type/subtype are emitted because it's not always clear.